release_tester/arangodb/starter/deployments/dc2dc.py

```diff
@@ -44,11 +45,32 @@ def _get_sync_status(cluster):
         raise Exception("could not fetch arangosync version from {url}, status code: {status_code}".
                         format(url=url, status_code=response.status_code))
 
-    status = response.json().get('status')
-    if not status:
-        raise Exception("missing status in response from {url}".format(url=url))
+    # Check the incoming status of the cluster.
+    incoming_status = STATUS_INACTIVE
+    shards = response.json().get('shards')
+    if shards and len(shards) > 0:
+        # It is the response from the target or proxy cluster.
+        incoming_status = response.json().get('status')
+        if not incoming_status:
+            raise Exception("missing incoming status in response from {url}".format(url=url))
 
-    return status
+    # Check the outgoing status of the cluster.
+    outgoing_status = STATUS_INACTIVE
+    outgoing = response.json().get('outgoing')
+    if outgoing and len(outgoing) > 0:
+        # It is the response from the source or proxy cluster.
+        outgoing_status = outgoing[0].get('status')
+        if not outgoing_status:
+            raise Exception("missing outgoing status in response from {url}".format(url=url))
```

Comment:
please also add the reply to the exception

Reply:
done